automation/src/test/scala/GoogleSpec.scala

```diff
@@ -1,11 +1,12 @@
 import org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec
-import org.scalatest.{FlatSpec, Matchers, ParallelTestExecution}
+import org.scalatest.{FlatSpec, Ignore, Matchers, ParallelTestExecution}
 import org.scalatest.selenium.WebBrowser
 
 /**
   * Example test for running with Selenium Grid until we work out docker
   * networking to be able to access FiaBitC.
   */
+@Ignore
```

Comment:
ignore `GoogleSpec`

Reply:
I think this whole test can go away. It was created as a first test of our selenium infrastructure. It doesn't even touch FireCloud.

Reply:
`GoogleSpec` is gone.